### ISSUE TITLE
Update dark theme palette

### ISSUE THREAD

**Issue by shinjith-dev** [dullage/flatnotes#265]
The overall color scheme looks great.

The editor's color scheme is unappealing in dark mode! Possibly because the text color blends too much with the background. And the toolbar visibility is unusually transparent; this could be due to inadequate support for dark mode in the editor library used, but I'm not sure.

If you don't want to waste your time on this, I'd be happy to raise a PR.

**Comment by dullage** [comment]
I agree in regard to the editor toolbar in dark mode. The trouble is, these icons (and their colour) are all part of a single image (see [here](https://github.com/nhn/tui.editor/blob/0c5c11bac0b9954e2cac2ff2a081ce8c80a5f14d/apps/editor/src/img/toastui-editor.png)). If you fancy playing around with the colours in this image (ideally sticking to the [theme colours](https://github.com/dullage/flatnotes/blob/develop/client/style.css)) then I could swap it out.

**Comment by shinjith-dev** [comment]
@dullage  I tried some workarounds, but since the icons do not accept color, even darker shades do not show much contrast. Here are some options we have. Since you insist on not modifying the palette, the final one is impractical.

(orginal)
![Firefox_Screenshot_2024-12-03T14-05-33 152Z](https://github.com/user-attachments/assets/a29978f7-8854-4774-b667-22d445813e89)

![Firefox_Screenshot_2024-12-03T14-00-04 679Z](https://github.com/user-attachments/assets/4e198f1b-58e9-4a9d-8403-5596a0b69efb)

![Firefox_Screenshot_2024-12-03T14-04-53 938Z](https://github.com/user-attachments/assets/95b54923-6d89-42e0-adc0-f1488a873ead)

![Firefox_Screenshot_2024-12-03T14-11-45 353Z](https://github.com/user-attachments/assets/80cf5505-dfb9-4d40-9c3e-b0d6ccdc7bbb)

This tool is useful if you're planning to update palette for dark mode [https://colorffy.com/dark-theme-generator](https://colorffy.com/dark-theme-generator)

**Comment by Code-Otto** [comment]
There's a dark theme example from toast-ui devs here: https://nhn.github.io/tui.editor/latest/tutorial-example06-dark-theme displaying a toolbar with bright icons

At a quick glance I see they're setting a `theme: 'dark'` attribute at the `Editor` object's constructor

**Comment by dullage** [comment]
Thanks @Code-Otto. Looking at their example made me realise that I wasn't using their dark mode icons! I thought I was but a number of versions ago I changed from using a `.dark-theme` class to a `.dark` one and had missed a few cases (including these buttons!).

I've pushed a fix, and it looks like this now:
![image](https://github.com/user-attachments/assets/855a6edd-a4e4-453e-b693-b914bfc1a313)
